lib/src/types.ts

```diff
@@ -95,8 +95,14 @@ export type AxisProps<
   xScale: ScaleLinear<number, number, never>;
   yScale: ScaleLinear<number, number, never>;
   font?: SkFont | null;
-  lineColor: Color | { grid: Color; frame: Color };
-  lineWidth: number | { grid: number; frame: number };
+  lineColor:
+    | Color
+    | { x: Color; y: Color; frame: Color }
+    | { grid: Color; frame: Color };
+  lineWidth:
+    | number
+    | { x: number; y: number; frame: number }
+    | { grid: number; frame: number };
```

Comment:
I debated whether we should make this exclusionary, in that if `grid` is included, then we don't accept `x` or `y`, and vice versa, but I thought this way was a bit more graceful in how it handled these properties.

Reply:
Throwing out options to consider everything. Thoughts on making the type something like:

```ts
lineColor:
  | Color
  | { grid: Color | {x: Color, y: Color}; frame: Color }
```

Reply:
Done. 🔥